### PR TITLE
update README for supported browsers & into text

### PR DESCRIPTION
**Pull request by ryestew**
fix https://github.com/ethereum/remix-project/issues/2229

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,18 +8,23 @@
...
 
-**Remix Project** is a platform for development tools that use a plugin architecture. It encompasses sub-projects including Remix Plugin Engine, Remix Libraries, and of course Remix IDE.
+## Remix IDE
+**Remix IDE** is used for the entire journey of contract development by users of any knowledge level. It fosters a fast development cycle and has a rich set of plugins with intuitive GUIs.  The IDE comes in 2 flavors and a VSCode extension:
 
-**Remix IDE** is an open source web and desktop application. It fosters a fast development cycle and has a rich set of plugins with intuitive GUIs. Remix is used for the **entire journey of contract development with [Solidity language](https://soliditylang.org/)** as well as a playground for learning and teaching [Ethereum](https://ethereum.org/).
+**Remix Online IDE**, see: [https://remix.ethereum.org](https://remix.ethereum.org)
 
-Start developing using Remix on browser, visit: [https://remix.ethereum.org](https://remix.ethereum.org)
+:point_right: Supported browsers: Firefox, Chrome, Brave. No support Remix's use on tablets or smart phones or telephones.
```
Comment (yann300) [inline README.md:18]: we should specify the version number too

Comment (Aniket-Engg) [inline README.md:18]: Either add `etc.` at the end of browsers list or add `and` before Brave.

Comment (Aniket-Engg) [inline README.md:18]: No support __for__ Remix's use on tablets, __smartphones__ or telephones

File: README.md [hunk 2]
```diff
@@ -8,18 +8,23 @@
...
-For desktop version, see releases: [https://github.com/ethereum/remix-desktop/releases](https://github.com/ethereum/remix-desktop/releases)
+**Remix Desktop IDE**, see releases: [https://github.com/ethereum/remix-desktop/releases](https://github.com/ethereum/remix-desktop/releases)
 
-![Remix screenshot](https://github.com/ethereum/remix-project/raw/master/apps/remix-ide/remix_screenshot.png)
+![Remix screenshot](https://github.com/ethereum/remix-project/raw/master/apps/remix-ide/remix-screenshot-400h.png)
 
-:point_right: **Remix libraries** work as a core of native plugins of Remix IDE. Read more about libraries [here](libs/README.md)
+**VSCode extension**, see: [Ethereum-Remix](https://marketplace.visualstudio.com/items?itemName=RemixProject.ethereum-remix)
+
+## Remix libraries**
```
Comment (Aniket-Engg) [inline README.md:26]: ** seems unnecessary

File: README.md (hunk 1 above)
Comment (ryestew) [reply README.md:18]: @yann300 - So in checking with Circle - we test with chrome v101.0.4951.64 and with FF v100.0.1. and Brave is Chromium - so what do we say about Brave?  And by specifying version num, then we need to update this readme every we use a newer version of the browsers in a CI.  Alternatively, we can say we test with the latest version of all Firefox & Chrome ( and leave out Brave).

File: README.md (hunk 2 above)
Comment (ryestew) [reply README.md:26]: @Aniket-Engg  Are you saying we should not tell people here about the VSCode plugin?  I mention it in 2 places.  I could take it out of both.

File: README.md (hunk 1 above)
Comment (yann300) [reply README.md]: we can update now the md to reflect the current version used in CI. For the moment let's not mention Brave.

**Review by bunsenstraat — APPROVED** [review]

**Review by Aniket-Engg — APPROVED** [review]

**Pull request merged**